static/js/components/requestgroup.vue

```diff
@@ -245,6 +261,13 @@
       }
     },
     watch: {
+      proposalOptions: function() {
+        if (this.hasRetrievedProposals && this.proposalOptions.length < 2) {
```

Comment:
why is this < 2?

Reply:
Because the first item in the proposals options list is always an empty option. I'll add a comment since that's not clear.